Security/Http/ResourceOwnerMapInterface.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware.Info <opensource@hardware.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HWI\Bundle\OAuthBundle\Security\Http;
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @author Sérgio Rafael Siqueira <sergio@inbep.com.br>
```

Comment:
In fact you are not the author of this code cause you extracted this code from another file, so the author should be set to Alexander.